editor/plugins/org.fusesource.ide.projecttemplates/src/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationProjectWizardRuntimeAndCamelPage.java

```diff
@@ -166,42 +168,68 @@ public void widgetSelected(SelectionEvent e) {
 		camelVersionLabel.setLayoutData(camelLabelData);
 		camelVersionLabel.setText(Messages.newProjectWizardRuntimePageCamelLabel);
 
-		camelVersionCombo = new Combo(camelGrp, SWT.RIGHT | SWT.READ_ONLY);
+		camelVersionCombo = new Combo(camelGrp, SWT.RIGHT | SWT.DROP_DOWN);
 		GridData camelComboData = new GridData(SWT.FILL, SWT.CENTER, true, false, 2, 1);
 		camelVersionCombo.setLayoutData(camelComboData);
-		camelVersionCombo.setItems(getSupportedCamelVersions());
-		camelVersionCombo.select(Math.max(camelVersionCombo.getItemCount()-1, 0));
+		camelVersionCombo.setItems(CamelCatalogUtils.getOfficialSupportedCamelCatalogVersions().toArray(new String[CamelCatalogUtils.getOfficialSupportedCamelCatalogVersions().size()]));
```

Comment:
for a more compact code, you can use CamelCatalogUtils.getOfficialSupportedCamelCatalogVersions().stream().toArray(String[]::new)